scripts/breadcrumbs.js

```diff
@@ -78,6 +75,11 @@ export default async function createBreadcrumbs(container) {
       container.append(renderBreadcrumb(crumb));
     }
   });
+}
+
+const breadcrumb = document.querySelector('.breadcrumb');
 
-  await breadCrumbsCSS;
+// check if breadcrumb div exists
+if (breadcrumb !== undefined) {
```

Comment:
`if (!breadcrumb)` is more common and readable unless you are explicitly setting the variable to undefined.